mod/register.php

```diff
@@ -1,9 +1,10 @@
 <?php
 /**
- * @file mod/register.php
+ * @file_tag_list_to_file mod/register.php
```

Comment:
What's this?

Reply:
An artifact from my IDE, I have no idea why. It happens sometimes when I regenerate the use statement list.